### PR TITLE
Update build for Task Role and Parameter Store.

### PR DESCRIPTION
**Pull request by Omicron7**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,17 +1,27 @@
 #!/bin/bash
 
+PG_PASSWORD="$(date +%s | sha256sum | base64 | head -c 16)"
+docker network create $PROJECT_NAME
+docker run --rm --network=$PROJECT_NAME --name=$PROJECT_NAME-redis -d redis
+docker run --rm --network=$PROJECT_NAME --name=$PROJECT_NAME-pg -e POSTGRES_PASSWORD=$PG_PASSWORD -d postgres:11
```
Comment (jbirdjavi) [inline build.sh:6]: Do you need to stop this container?

File: build.sh [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
     --build-arg DD_API_KEY=$DD_API_KEY \
+    --build-arg REDIS_PORT_6379_TCP_ADDR=$PROJECT_NAME-redis \
+    --build-arg REDIS_PORT_6379_TCP_ADDR_SESSION=$PROJECT_NAME-redis \
+    --build-arg DB_ENV_POSTGRESQL_USER=postgres \
+    --build-arg DB_ENV_POSTGRESQL_PASS=$PG_PASSWORD \
+    --build-arg DB_PORT_5432_TCP_ADDR=$PROJECT_NAME-pg \
     -t 056154071827.dkr.ecr.us-east-1.amazonaws.com/$PROJECT_NAME:$ENVIRONMENT-$BUILD_NUMBER .
 rc=$?
 
+docker stop $PROJECT_NAME-redis $PROJECT_NAME-pg
```
Comment (Omicron7) [inline build.sh:22]: It stops both here.

File: build.sh (hunk 1 above)
Comment (Omicron7) [reply build.sh:6]: Line 22 will stop both containers.

File: build.sh (hunk 2 above)
Comment (jbirdjavi) [reply build.sh:22]: Ahh yes.  Need to look closer.  :)

**Pull request merged**